openslides/motions/projector.py

```diff
@@ -70,7 +70,9 @@ def get_requirements(self, config_entry):
         else:
             yield motion_block
             yield motion_block.agenda_item
-            yield from motion_block.motion_set.all()
+            for motion in motion_block.motion_set.all():
+                yield motion
+                yield motion.state.workflow
```

Comment:
We do not need to send the workflows multiple times (once per motion) but only once at all.